packages/yum/groonga.spec.in

```diff
@@ -15,7 +15,7 @@ URL:		http://groonga.org/
 Source0:	http://packages.groonga.org/source/groonga/groonga-%{version}.tar.gz
 
 BuildRoot:	%{_tmppath}/%{name}-%{version}-%{release}-%(%{__id_u} -n)
-BuildRequires:	arrow-devel = @APACHE_ARROW_VERSION@
+BuildRequires:	arrow-devel = @APACHE_ARROW_VERSION@, arrow-devel < @APACHE_ARROW_NEXT_VERSION@
```

Comment:
Is this condition right?
`,` means `AND`, doesn't it?

```suggestion
BuildRequires:	arrow-devel >= @APACHE_ARROW_VERSION@, arrow-devel < @APACHE_ARROW_NEXT_VERSION@
```

Reply:
This is a mistake.
I have fixed it.